Detectors/MUON/MCH/Raw/Common/src/SampaCluster.cxx

```diff
@@ -21,16 +21,18 @@ namespace mch
 namespace raw
 {
 
-SampaCluster::SampaCluster(uint16_t timestamp, uint32_t chargeSum)
-  : timestamp(impl::assertIsInRange("timestamp", timestamp, 0, 0x3FF)),   // 10 bits
+SampaCluster::SampaCluster(uint16_t timestamp, uint32_t bunchCrossing, uint32_t chargeSum)
+  : timestamp(impl::assertIsInRange("timestamp", timestamp, 0, 0x3FF)),
+    bunchCrossing(impl::assertIsInRange("bunchCrossing", bunchCrossing, 0, 0xFFFFF)),
```

Comment:
if the bunchcrossing is only 20 bits, that should be asserted here

Reply:
I though it would be asserted by
```
impl::assertIsInRange("bunchCrossing", bunchCrossing, 0, 0xFFFFF)
```
Did I miss something?